src/property-filter/internal.tsx

```diff
@@ -314,7 +315,13 @@ const PropertyFilterInternal = React.forwardRef(
 
     return (
       <div {...baseProps} className={clsx(baseProps.className, styles.root)} ref={mergedRef}>
-        <div className={clsx(styles['search-field'], analyticsSelectors['search-field'])}>
+        <div
+          className={clsx(
+            styles['search-field'],
+            analyticsSelectors['search-field'],
+            !fullWidth && styles['limited-width']
```

Comment:
To be consistent, can we use the same `full-width` as a class rather than negating the condition and using `limited-width`?

```tsx
fullWidth && styles['full-width']
```

```scss
.search-field {
  max-inline-size: calc(#{styles.$breakpoint-x-small} - 2 * #{awsui.$space-l});

  &.full-width {
    max-inline-size: none;
  }
}
```